docs/source/trainer.rst

```diff
@@ -9,7 +9,7 @@ Trainer
    :exclude-members:
         run_pretrain_routine,
         _abc_impl,
-        _Trainer__set_random_port,
+        _Trainer_set_random_port,
         _Trainer__set_root_gpu,
         _Trainer__init_optimizers,
         _Trainer__parse_gpu_ids,
```

Comment:
```suggestion
```
can be removed as non of them is presented in the docs

Reply:
they're not presented in the docs because they are listed here lol...

Reply:
even they are not listed here I have not found them in docs